<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB传输工具</string>
  </property>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SearchButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>状态显示</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ConnectButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>CloseButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PathEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>text.dat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
